# BCB330-CCG-lab

## Functionally characterizing the effects of multiple mutations

## Abstract

Predicting the effects of gene mutations is vital to further understand the pathogenicity of genetic variants. While single mutations can significantly disrupt gene function, multiple mutations within the same gene often interact epistatically, leading to complex effects on gene activity. Multiple mutations (MMs) are defined as two or more non-synonymous mutations within the same gene, which can be classified as either cis or trans. Trans MMs occur on distinct alleles of the gene, while cis MMs reside on the same allele. The cis configuration frequently results in intragenic epistasis, where the combined effects of mutations deviate from simple additivity, often manifesting as multiplicative, compensatory, or other non-linear interactions.These interactions can profoundly impact the fitness and evolutionary trajectory of the gene and its product. In the context of cancer, studying multiple mutations provides valuable insights into their roles in tumor growth and evolution. Despite the growing recognition of the functional relevance of cis MMs, current approaches to characterizing their effects lack scalability and broad applicability. To address these limitations, we have developed a computational workflow that leverages biophysical and evolutionary approaches to systematically characterize the effects of cis MMs. This scalable pipeline is designed for genome-wide application, enabling the analysis of multiple mutations across diverse genes. In cancer research, our workflow can uncover mutational landscapes of oncogenes and tumor suppressor genes (TSGs), identifying potential pathogenic cis MMs as putative biomarkers. This approach offers a powerful tool for advancing our understanding of dynamics of multiple mutations in cancer and other genetic disorders.



### Project description

Epistasis, a genetic phenomenon where the effect of a gene mutation depends on the presence or absence of mutations in other genes, serves as a crucial concept in understanding complex biological interactions. Extending this idea to cancer research, our project aims to characterize and quantify the effects of cis multiple mutations—two or more nonsynonymous mutations occurring within the same gene—in the context of cancer. A study has shown that approximately one in four tumors harbor multiple mutations within a cancer-associated gene, highlighting the significance of this phenomenon in tumorigenesis (Gorelick et al. 2020). To characterize these mutations,weI will explore and evaluate existing computational biology tools, including biophysical tools like DDMut and DynaMut2, and evolutionary tools like HMMvar. This evaluation will focus on assessing their efficacy, scalability, and applicability for characterizing cis multiple mutations.We will also experiment with a coevolutionary approach using Direct Coupling Analysis (DCA) to uncover potential interactions between mutations and how they may contribute to cancer progression. Finally, we will apply these tools to analyze mutations identified in existing literature on multiple mutations in cancer, as well as potentially other datasets, to identify and understand patterns of interest that may provide insights into cancer biology and therapeutic strategies.

References 

- Gorelick AN, Sánchez-Rivera FJ, Cai Y, Bielski CM, Biederstedt E, Jonsson P, Richards AL, Vasan N, Penson AV, Friedman ND, et al. 2020. Phase and context shape the function of composite oncogenic mutations. Nature. 582(7810):100–103. doi:https://doi.org/10.1038/s41586-020-2315-8. [accessed 2024 Aug 19]. https://www.nature.com/articles/s41586-020-2315-8#Sec1.

## License

Distributed under the MIT License. See `LICENSE.txt` for more information.




<!-- CONTACT -->
## Contact

Arhan Rupani - arhan.rupani@mail.utoronto.ca

Project Link: 



<!-- ACKNOWLEDGMENTS -->
## Acknowledgments
